WcaOnRails/app/assets/javascripts/application.js

```diff
@@ -172,18 +167,12 @@ $(function() {
   $('input.wca-autocomplete').wcaAutocomplete();
 
   var $tablesToFloatHeaders = $('table.floatThead');
-  if (!isMobile()) {
-    $tablesToFloatHeaders.floatThead({
-      zIndex: 999, // Allow bootstrap popups (z-index 1000) to show up on top.
-    });
-    // Workaround for https://github.com/mkoryak/floatThead/issues/263
-    $tablesToFloatHeaders.each(function() {
-      var $table = $(this);
-      $table.closest('.table-responsive').scroll(function(e) {
-        $table.floatThead('reflow');
-      });
-    });
-  }
+  $tablesToFloatHeaders.floatThead({
```

Comment:
I tested it in my phone, and it does seem to work!

On Tue, Mar 29, 2016, 04:25 Pedro Santos <notifications@github.com> wrote:

> In WcaOnRails/app/assets/javascripts/application.js
> <https://github.com/cubing/worldcubeassociation.org/pull/442#discussion_r57707752>
> :
>
> > @@ -172,18 +167,12 @@ $(function() {
> >    $('input.wca-autocomplete').wcaAutocomplete();
> >
> >    var $tablesToFloatHeaders = $('table.floatThead');
> > -  if (!isMobile()) {
> > -    $tablesToFloatHeaders.floatThead({
> > -      zIndex: 999, // Allow bootstrap popups (z-index 1000) to show up on top.
> > -    });
> > -    // Workaround for https://github.com/mkoryak/floatThead/issues/263
> > -    $tablesToFloatHeaders.each(function() {
> > -      var $table = $(this);
> > -      $table.closest('.table-responsive').scroll(function(e) {
> > -        $table.floatThead('reflow');
> > -      });
> > -    });
> > -  }
> > +  $tablesToFloatHeaders.floatThead({
>
> Does it work on mobile too now?
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/cubing/worldcubeassociation.org/pull/442/files/f38b91da8c8da18896b111ab7522aa33b1dd8ca9#r57707752>
>